specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-03-preview/examples/ContainerServiceListOrchestrators.json

```diff
@@ -0,0 +1,207 @@
+{
+  "parameters": {
+    "api-version": "2022-08-03-preview",
+    "subscriptionId": "subid1",
+    "location": "location1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid1/providers/Microsoft.ContainerService/locations/location1/orchestrators",
+        "name": "default",
+        "type": "Microsoft.ContainerService/locations/orchestrators",
+        "properties": {
+          "orchestrators": [
+            {
+              "orchestratorType": "DCOS",
```

Comment:
I don't think we return DCOS, Swarm, DockerCE anymore. Should remove those in the sample

Reply:
Sure, updated.